### ISSUE TITLE
Accessibility Checklists: Analyze and synthesize findings from prototype test sessions

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-team#318]
I have done some analysis and created preliminary findings documents that I shared with the team.

- [Preliminary findings 1](https://docs.google.com/document/d/1fMXbs4YS5PwllSmG-chz-tSUKKj7c_S_CapdJlMmmFc/edit?usp=sharing) (results after we tested with 3 participants. We made changes to the prototype based on feedback and tested with the remaining 3 participants)
- [Preliminary findings 2](https://docs.google.com/document/d/17cZffXESqUoHgyxU_SekbaPAgOC9N8o734LBO7X9VBs/edit?usp=sharing)

**Comment by jaclinec** [comment]
Findings were presented to the team in our last CS UX sync and we discussed next steps. 

- Sarah will come up with some ideas and a mockup to better communicate the page purpose at the top of the page that is more visual and reduces text. 
- Amy L will create some options for changes to the main Accordion page that help communicate conformance and guide users to the checklist page. 

We did not decide on a page title yet. We'll continue to iterate the design and may possibly ask for more feedback from users asynchronously if we feel it is needed before launch.

**Comment by jaclinec** [comment]
Analysis and synthesis is complete for this research project. I've presented findings to the team and we continue to iterate on the prototype design based on user feedback.

Documentation:

- [ Analysis document](https://docs.google.com/document/d/15rdiX2SAcX5vL6yOLCyNNjatGMTk8ExWRcuW9M0PI6w/edit?usp=sharing)
- [Preliminary findings 1 ](https://docs.google.com/document/d/1fMXbs4YS5PwllSmG-chz-tSUKKj7c_S_CapdJlMmmFc/edit?usp=sharing)(The early findings after we'd tested with 3 participants. We made changes to the prototype and tested with 3 more)
- [Final findings report](https://docs.google.com/document/d/17cZffXESqUoHgyxU_SekbaPAgOC9N8o734LBO7X9VBs/edit?usp=sharing) (The final report incorporating findings from all the sessions. The report was kept simple in the interest of reporting findings quickly to the team so we could be agile and make fast changes to the prototype before launch) 

After these were shared and discussed, we also reached out to the VA and colleagues from 10x for more feedback on our latest prototype updates. We continue to make changes and improvements and plan to launch the page before January monthly call.
